ajax/libs/html5tooltipsjs/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "html5tooltipsjs",
+  "version": "1.7.0",
+  "description": "Tooltips with smooth 3D animation.",
+  "main": "html5tooltips.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ytiurin/html5tooltipsjs.git"
+  },
+  "keywords": [
+    "tooltips",
+    "tooltip",
+    "animation",
+    "html5",
+    "3d",
+    "ui"
+  ],
+  "author": "Yevhen Tiurin",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ytiurin/html5tooltipsjs/issues"
+  },
+  "homepage": "https://github.com/ytiurin/html5tooltipsjs",
+  "npmName": "html5tooltipsjs",
+  "npmFileMap": [
+    {
+      "basePath": "/",
```

Comment:
@maruilian11 make you review comment clean and clear please.

@inlutero we should use `"basePath": "",` in this case, thanks.